test/acceptance/e2e.prepare.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+const util = require('../utils/common');
+
+const files = util.loadFiles(path.join(__dirname, 'e2e'));
+const template = util.loadTemplate(path.join(__dirname, 'e2e.html.tpl'));
+
+files.reduce((promise, file) => {
+    return promise.then(() => util.takeReference(file, template, true));
```

Comment:
I dont understand the need of the `promise.then()`

Reply:
why not just? 
```javascript 
files.forEach(file => util.takeReference(file, template, true));
```

Reply:
This is just to execute the `takeReference` in a sequence (promise chain)

Reply:
👍